### PR TITLE
Blog - Dec 2023

### PR DESCRIPTION
**Pull request by RossKen**
### Type of PR

- [ ] BUG
- [ ] FEAT
- [ ] MAINT
- [X] DOC

### PR Checklist

- [ ] Added documentation for changes
- [ ] Added feature to example notebooks or tutorial (if appropriate)
- [ ] Added tests (if appropriate)
- [ ] Updated CHANGELOG.md (if appropriate)
- [ ] Made changes based off the latest version of Splink
- [ ] Run the [linter](https://moj-analytical-services.github.io/splink/dev_guides/changing_splink/lint.html)

### CODE REVIEW

File: docs/blog/posts/2023-12-06-feature_update.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+```bsh
+pip install 'splink[spark]'
+```
+
+See the [Getting Started page](../../getting_started.md#backend-specific-installs) for further guidance.
+
+## :no_entry_sign: Drop support for python 3.7
+
+From Splink 3.9.7, support has been dropped for python 3.7. This decision has been made to managing dependency clashes in the back end of Splink.
```
Comment (RobinL) [inline docs/blog/posts/2023-12-06-feature_update.md:104]:
```suggestion
From Splink 3.9.7, support has been dropped for python 3.7. This decision has been made to manage dependency clashes in the back end of Splink.
```

**Review by RobinL — APPROVED** [review]
LGTM 👍

File: docs/blog/posts/2023-12-06-feature_update.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+## :no_entry_sign: Drop support for python 3.7
+
+From Splink 3.9.7, support has been dropped for python 3.7. This decision has been made to manage dependency clashes in the back end of Splink.
+
+If you are working with python 3.7, please revert to Splink 3.9.6.
+
+```bsh
+pip install splink==3.9.6
+```
```
Comment (ThomasHepworth) [inline docs/blog/posts/2023-12-06-feature_update.md:110]: I know I mentioned that we support some version of python 3.7 (>=3.7.1), but I think we'll have to upgrade to 3.8 soon anyway, so this section looks good.

**Review by ThomasHepworth — APPROVED** [review]
Thanks Ross, this looks great.

**Pull request merged**